android/services/mongodb-remote/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/remote/internal/OkHttpInstrumentedTransportUnitTests.kt

```diff
@@ -0,0 +1,85 @@
+package com.mongodb.stitch.android.services.mongodb.remote.internal
+
+import okhttp3.Call
+import okhttp3.Connection
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+import java.io.IOException
+import java.util.concurrent.TimeUnit
+
+open class ChainImpl: Interceptor.Chain {
```

Comment:
[q] why are we not implementing any of these yet?

Reply:
We don't need to, this class is only for mocking.

Reply:
ah kk, can we replace the `TODO`s then with some sort of fatal error?

Reply:
Sure

Reply:
[nit] add a comment explaining that this class is only here for mocking, and that the implementations are not necessary